utils/appliance/__init__.py

```diff
@@ -73,19 +74,42 @@ def run(self, ap_cli_command):
             "appliance_console_cli {}".format(ap_cli_command))
 
     def set_hostname(self, hostname):
-        self.run("-H {}".format(hostname))
+        self.run("-H {host}".format(host=hostname))
```

Comment:
**REQUIRED:** Looking at this now, I really think we should change ```.run()``` to be ```._run()```. There shouldn't be a reason for the average user to run this method. So let's make it less obvious.